code/game/machinery/buttons.dm

```diff
@@ -11,8 +11,7 @@
 	var/id = null
 	var/initialized_button = 0
 	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 50, BOMB = 10, BIO = 100, FIRE = 90, ACID = 70)
-	use_power = IDLE_POWER_USE
-	idle_power_usage = 2
+	idle_power_usage = MIN_IDLE_POWER_USAGE * 0.02
```

Comment:
what's goin on here?

Reply:
for use_power refer to the above comment
the rest is just 100 * 0.02 = 2 
so that buttons will draw a small 2 W